src/communication/azext_communication/manual/commands.py

```diff
@@ -7,16 +7,46 @@
 from azext_communication.manual._client_factory import cf_communication_identity
 from azext_communication.manual._client_factory import cf_communication_sms
 from azext_communication.manual._client_factory import cf_communication_phonenumbers
+from azext_communication.manual._client_factory import cf_communication_chat
 
 
 def load_command_table(self, _):
 
-    with self.command_group('communication identity', client_factory=cf_communication_identity) as g:
-        g.communication_custom_command('issue-access-token', "issue_access_token", client_factory=cf_communication_identity)
+    identity_arguments = ['connection_string']
+    with self.command_group('communication identity', client_factory=cf_communication_identity, is_preview=True) as g:
+        g.communication_custom_command('create-user', "communication_identity_create_user", identity_arguments, client_factory=cf_communication_identity)
+        g.communication_custom_command('delete-user', "communication_identity_delete_user", identity_arguments, client_factory=cf_communication_identity)
+        g.communication_custom_command('issue-access-token', "communication_identity_issue_access_token", identity_arguments, client_factory=cf_communication_identity)
+        g.communication_custom_command('revoke-access-tokens', "communication_identity_revoke_access_tokens", identity_arguments, client_factory=cf_communication_identity)
+        g.communication_custom_command('get-token-for-teams-user', "communication_identity_get_token_for_teams_user", identity_arguments, client_factory=cf_communication_identity)
 
-    with self.command_group('communication sms', client_factory=cf_communication_sms) as g:
-        g.communication_custom_command('send-sms', 'communication_send_sms')
+    sms_arguments = ['connection_string']
+    with self.command_group('communication sms', client_factory=cf_communication_sms, is_preview=True) as g:
+        g.communication_custom_command('send-sms', 'communication_send_sms', sms_arguments)
 
-    with self.command_group('communication phonenumbers', client_factory=cf_communication_phonenumbers) as g:
-        g.communication_custom_command('list-phonenumbers', 'communication_list_phonenumbers')
-        g.communication_custom_command('show-phonenumber', 'communication_show_phonenumber')
+    phonenumber_arguments = ['connection_string']
+    with self.command_group('communication phonenumbers', client_factory=cf_communication_phonenumbers, is_preview=True) as g:
+        g.communication_custom_command('list-phonenumbers', 'communication_list_phonenumbers', phonenumber_arguments)
+        g.communication_custom_command('show-phonenumber', 'communication_show_phonenumber', phonenumber_arguments)
```

Comment:
Why this command group changed form GA to Preview again?

Reply:
Changed back to GA.